kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -183,12 +183,10 @@
           // Instantiate the Vue instance directly using the Kolibri Vue constructor.
           this.contentView = new this.Kolibri.lib.vue(options); // eslint-disable-line new-cap
 
-          // ====== disable content logging until we track down some issues =======
-
-          // this.contentView.$on('startTracking', this.wrappedStartTracking);
-          // this.contentView.$on('stopTracking', this.wrappedStopTracking);
-          // this.contentView.$on('progressUpdate', this.wrappedUpdateProgress);
-          // this.initSession(this.Kolibri, this.channelId, this.contentId, this.kind);
+          this.contentView.$on('startTracking', this.wrappedStartTracking);
```

Comment:
This tracking code was previously disabled, and I'm re-enabling it here so that I have actual data to export. So, it's not directly used for CSV export itself, but it's ready to be re-enabled, and needed in order to have actual data to use, so I enable it again here.